scripts/create-icon.ts

```diff
@@ -40,22 +40,29 @@ const createElement = async (name: string, node: unknown) => {
 
   const jsxString = await svgToJsx(node as string);
 
+  const size = (key: string) => `${key} ?? size`;
+
   const updatedJsxString = jsxString.replace(
     /<svg([^>]*)>/,
-    `<svg$1 {...props} ref={ref}>`,
+    `<svg$1 width={${size("width")}} height={${size("height")}} ref={ref} {...rest}>`,
```

Comment:
can you explain this please?

Reply:
There is the options to pass width or height, if not passed use size as the default value